### PR TITLE
[1/x] Create request model

### PR DESCRIPTION
**Pull request by katesliang**
## Overview

Create request model

## Changes Made

- Define request model and its features

## Test Coverage

N/A

## Next Steps

Implementing request serializer in next PR

## Related PRs and Issues

#20

### CODE REVIEW

File: src/ride/controllers/create_ride_controller.py [hunk 1]
```diff
@@ -50,14 +50,26 @@ def process(self):
             return failure_response("Creator does not exist")
         creator_person = Person.objects.get(id=creator)
 
-        # Create path and ride and return new ride with given fields
-        path = Path.objects.create(
+        # Create new path or retrieve existing path
+        path_exists = Path.objects.filter(
```
Comment (chalo2000) [inline src/ride/controllers/create_ride_controller.py]: this was in the previous PR?

**Review by chalo2000 — CHANGES_REQUESTED** [review]
Can you explain what you did to change the request model? Also, why am I seeing changes here from a previous PR?

File: src/request/models.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from person.models import Person
+from ride.models import Ride
+
+
+class Request(models.Model):
+    ride = models.ForeignKey(Ride, on_delete=models.CASCADE)
```
Comment (chalo2000) [inline src/request/models.py:7]: many requests belong to one ride? how does this work

**Review by chalo2000 — CHANGES_REQUESTED** [review]
PR is bare, we're creating the models and need to understand what the bigger picture is. To me it is clear why an approvee/approver can have many requests and it also makes sense why a ride may have many requests but I don't know how this works; will there be a way to handle requests for a ride when it already happened, will the requests be deleted. I need more context somewhere and if it's not appropriate to put all the details on this PR (which I don't think it would since this is just creating models and nothing to do with functionality) then I need some issue that describes in a little more depth how requests comes into play.

File: src/request/models.py (hunk 2 above)
Comment (katesliang) [reply src/request/models.py:7]: ya, each ride can have many requests. created an issue #20 that explains the process further.

**Review by chalo2000 — APPROVED** [review]
Add under the Related PRs section the issue you created (#20) and then you can merge

**Review by joycrwu — APPROVED** [review]
cool

**Pull request merged**